jupyterhub/values.yaml

```diff
@@ -169,6 +170,8 @@ proxy:
     extraVolumeMounts: []
     extraStaticConfig: {}
     extraDynamicConfig: {}
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
```

Comment:
This is technically new since it wasn't defined before for `traefik` correct? Maybe that's fine?

Reply:
True, its fine - I'll update the topic of the PR to ensure i remember this in the changelog.